hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/x509/package-info.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+
+/**
+ * This package contains common routines used in creating an x509 based identity
+ * framework for HDDS.
+ */
+package org.apache.hadoop.hdds.security.x509;
+/*
+
+Architecture of Certificate Infrastructure for SCM.
+====================================================
```

Comment:
I think this should be a javadoc comment on the package, otherwise most people will never see it.  At least I didn't know about it until now. ;)

Also, I don't think we should duplicate this description in yet another place, it's already present in:

```
hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/package-info.java
hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/package-info.java
```

It should be in `common/src/main` only, and the others should refer to it.

Reply:
Actually this description is somewhat the tldr of the original design document in HDDS-4. Parts of it is obsolete, some parts were never implemented. I decided to remove this, as with the current and recent changes we'd better write a solid documentation page rather than trying to summarize this in the APIDoc I believe.

I removed the duplicated package-info file form the hdds/framework module.